examples/bicluster/plot_bicluster_newsgroups.py

```diff
@@ -68,7 +68,7 @@ def build_tokenizer(self):
 newsgroups = fetch_20newsgroups(categories=categories)
 y_true = newsgroups.target
 
-vectorizer = NumberNormalizingVectorizer(stop_words='english', min_df=5)
+vectorizer = NumberNormalizingVectorizer(min_df=5)
```

Comment:
Should use max_df if we previously had stop words

Reply:
Results with max_df = 0.4

`Vectorizing...
Coclustering...
Done in 5.51s. V-measure: 0.4124
MiniBatchKMeans...
Done in 10.31s. V-measure: 0.3657`